packages/beacon-node/src/network/reqresp/beaconBlocksMaybeBlobsByRoot.ts

```diff
@@ -98,13 +99,13 @@ export async function unavailableBeaconBlobsByRoot(
 
   // check and see if all blobs are now available and in that case resolve availability
   // if not this will error and the leftover blobs will be tried from another peer
-  const allBlobs = getBlockInputBlobs(blobsCache);
-  const {blobs, blobsBytes} = allBlobs;
```

Comment:
not related to this PR, we should be able to have blobBytes in this flow instead of null, could you add a TODO for it @g11tech ?